app/components/organisms/Footer/FooterMenuRight.tsx

```diff
@@ -9,14 +9,14 @@ const FooterMenuRight = ({
 	altText = 'Our company logo',
 }: FooterProps) => {
 	return (
-		<footer className="bg-secondary lg:py-16 dark:bg-dark-secondary">
-			<div className="container items-center justify-between border-b border-muted-foreground/75 py-8 lg:flex dark:border-dark-muted-foreground/75">
+		<footer className="dark:bg-dark-secondary bg-secondary lg:py-16">
+			<div className="dark:border-dark-muted-foreground/75 container items-center justify-between border-b border-muted-foreground/75 py-8 lg:flex">
 				<Link to="/" className="flex w-20 items-center justify-center lg:w-24">
 					<img src={logo} alt={altText} />
```

Comment:
See if you add your custom logo to the footer. Check the [tutorial](https://gedsargent.github.io/code-monster/guides/epic-news/ui/import-image-to-navbar-and-footer/#importing-the-image) at Code Monster for help with this.